code/menuui/optionsmenu.cpp

```diff
@@ -707,8 +708,13 @@ void options_button_pressed(int n)
 		case ABORT_GAME_BUTTON:
 			gamesnd_play_iface(InterfaceSounds::USER_SELECT);
 			choice = popup( PF_NO_NETWORKING | PF_BODY_BIG, 2, POPUP_NO, POPUP_YES, XSTR( "Exit Game?", 374));
-			if ( choice == 1 )
+			if ( choice == 1 ) {
+				if (gameseq_get_state(1) == GS_STATE_DEBRIEF && (Game_mode & GM_CAMPAIGN_MODE)) {
```

Comment:
The only way this call could be problematic is if there are no previous game states, which isn't possible in this case.